Configuration/TCA/tx_styleguide_flex.php

```diff
@@ -811,6 +811,42 @@
             ],
         ],
 
+        'flex_8' => [
```

Comment:
this entire field sholud go to elements_rte instead. elements_rte comes with a flex section already, another flex field should be added and the ts config setting above restricted to that field only.